src/ingestion/connectors/git/bitbucket-cloud/source_bitbucket_cloud/streams/base.py

```
@@ -0,0 +1,186 @@
+    if resp.status_code != 403:
+        return False
+    if resp.headers.get("Retry-After"):
+        return True
+    if resp.headers.get("X-RateLimit-Remaining") == "0":
+        return True
+    try:
+        body_text = resp.text.lower()
+        if "rate limit" in body_text:
+            return True
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)